app/pb_kits/playbook/pb_dashboard/pbChartsLightTheme.js

```diff
@@ -144,6 +144,20 @@ const highchartsTheme = {
       threshold: null,
     },
 
+    // GAUGE STYLES
+    // solidgauge: {
+    //   dataLabels: {
+    //     // borderWidth: 0,
+    //     color: colors.text_lt_default,
+    //     // enabled: true,
+    //     style: {
+    //       fontFamily: typography.font_family_base,
+    //       fontWeight: typography.bold,
+    //       fontSize: typography.heading_4,
+    //     },
+    //   },
+    // },
```

Comment:
What's the plan here

Reply:
Managed to get this sorted.  Was a silly order of operations issue.  Moved gauge formatting/style options over into pbChartsLightTheme.